pipelines/tekton/aiedge-e2e/examples/tensorflow-housing.trigger-template.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: TriggerTemplate
+metadata:
+  name: tensorflow-housing
+spec:
+  params:
```

Comment:
A TriggerTemplate just wraps around a resource such as a PipelineRun, and provides values for the PipelineRun params. These all have default values here, then a smaller number of them will be overridden by the TriggerBinding, which will take info from the webhook payload and populate the relevant TriggerTemplate params with it.